packages/cli/test/unit/util/deploy/create-git-meta.test.ts

```diff
@@ -277,13 +277,6 @@ describe('createGitMeta', () => {
         `Failed to get last commit. The directory is likely not a Git repo, there are no latest commits, or it is corrupted.`
       );
 
-      // skip next line
-      await lines.next();
-
-      line = await lines.next();
-      expect(line.value).toContain(
-        `Failed to determine if Git repo has been modified:`
-      );
```

Comment:
This output doesn't happen anymore. It seems better to only render out the most relevant error if both of these errors occur.

It's not clear how to reproduce this second issue without triggering the first. If anyone can think of one, I'll add another test for it.